buildSrc/src/main/groovy/io/micronaut/guides/GuideAsciidocGenerator.groovy

```diff
@@ -139,6 +140,37 @@ class GuideAsciidocGenerator {
             text = text.replace("@minJdk@", metadata.minimumJavaVersion?.toString() ?: "1.8")
             text = text.replace("@api@", 'https://docs.micronaut.io/latest/api')
 
+             text = text.replaceAll(~/@(\w*):?features@/) { List<String> matches ->
+
+                 String app = matches[1] ?: 'default'
```

Comment:
can you extract a method so that this code is not duplicated: 

```
 String app = matches[1] ?: 'default'
List<String> features = metadata.apps.find{ it.name == app }.features
if(guidesOption.language == Language.GROOVY) {
    features -= 'graalvm'
}
```

Reply:
I have extracted methods